docs/nl/docs/index.md

```diff
@@ -403,19 +403,19 @@ Terugkomend op het vorige code voorbeeld, **FastAPI** zal:
 
 Dit was nog maar een snel overzicht, maar je zou nu toch al een idee moeten hebben over hoe het allemaal werkt.
 
-Probeer de lijn te veranderen met:
+Probeer de regel te veranderen met:
```

Comment:
ah deze gemist, het is: "veranderen naar:"

Reply:
(zie ook regel 418 die wel goed is ;)

Reply:
Dit is een lastige, wat er bedoeld wordt is "De regel _met_ het volgende erin (wat op L409 staat), probeer die te veranderen ... van ... naar. Zie je? In het Engels is het 
> Try changing the line with ... from ... to

maar in het Nederlands is dit lastig door het omkeren van onderwerp en werkwoord 🤔

Reply:
`Probeer deze regel te veranderen: ...   van  .... naar ...`